Cab share/Login/LoginView.swift

```diff
@@ -101,7 +116,31 @@ struct LoginView: View {
             }
             .navigationBarHidden(true)
             .preferredColorScheme(.dark)
+//            .onAppear{
```

Comment:
What is this commented code means?

Reply:
onAppear part? is one of different ways to have information about what view show, but I did in a different way so I will delete it

Reply:
I resolve this as soon as you delete )